README.md

```diff
@@ -11,11 +11,18 @@
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3680/badge)](https://bestpractices.coreinfrastructure.org/projects/3680)
 <!-- markdown-link-check-enable -->
 
-> **Note:**
->
-> This readme and related documentation are Work in Progress.
+Chaos Mesh is an open source cloud-native Chaos Engineering platform. It offers various types of fault simulation and has an enormous capability to orchestrate fault scenarios.
 
-Chaos Mesh is a [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) hosted project. It is a cloud-native Chaos Engineering platform that orchestrates chaos on Kubernetes environments. At the current stage, it has the following components:
+Using Chaos Mesh, you can conveniently simulate various abnormalities that might occur in reality during the development, testing, and production environments and find potential problems in the system. To lower the threshold for a Chaos Engineering project, Chaos Mesh provides you with a visualization operation. You can easily design your Chaos scenarios on the Web UI and monitor the status of Chaos experiments.
+
+![cncf_logo](./static/cncf.png#gh-light-mode-only)
+![cncf_logo](./static/cncf-white.png#gh-dark-mode-only)
```

Comment:
I found another problem. Our logo also needs to suit the dark mode.

Update it in this PR as well? Copy this image (https://github.com/chaos-mesh/chaos-mesh/blob/master/ui/app/src/images/logo-white.svg) to static and update it.

Reply:
Done